ocaml/testsuite/tests/statmemprof/stop_start_in_callback.ml

```diff
@@ -28,7 +31,7 @@ module AllocSet = Set.Make(Int3Tuples)
 (* A combined 7-block 33-word allocation *)
 
 let[@inline never] f33 n =
-  ((n, n, (n, n, n, (n,n,n,n,n))), (n, n, (n, n, n, (n,n,n,n,n))))
+  ((n, n, (n, n, n, (n,n,n,n,n))), (n, n, (n, n, n, (n,n,n,n,0))))
```

Comment:
(I don't see how the change could be
a problem, but am curious why it was
applied.)

Reply:
I think it's to avoid a case where flambda2 can see two block allocations are the same.